playbooks/provisioning/openstack/README.md

```diff
@@ -67,6 +67,9 @@ The `openstack_num_masters`, `openstack_num_infra` and
 `openstack_num_nodes` values specify the number of Master, Infra and
 App nodes to create.
 
+The `openstack_node_removal_policies` allows you to specify which App nodes to
+remove.
```

Comment:
So when I first tried this, I put in the names of the servers (e.g. `app-node-1.openshift.example.com`) and that didn't do anything. What needs to go here is the numerical index (e.g. `1`) of the node.

Can you update the description here to make that clear (and maybe give an example?)

Eventually, I'd like us to support the hostnames as well, but this is fine for now.